client/src/components/Bio/Bio.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from "react";
+import "./Bio.css";
+import Icon from "../Icon/Icon";
+import { Link } from "react-router-dom";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEdit } from "@fortawesome/free-solid-svg-icons";
+import InputTextField from "components/InputTextField/InputTextField";
+
+library.add(faEdit);
+// Class Based React Component
+class Bio extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props);
+
+    // Default CSS class to apply to the Component
+    this.state = {
+      classList: "Bio",
+      isUser: this.props.isUser
```

Comment:
Is this going to change? It doesn't look like it does. If it's not, and it's already coming in through `props`, then can you keep it props and turn this into a functional component?